Vagrantfile

```diff
@@ -79,6 +79,24 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  ## Provider-specific settings, libvirt
+  config.vm.provider :libvirt do |v,override|
+    override.vm.box = "fedora21"
+    override.vm.box_url = "http://citozin.com/fedora21.box"
```

Comment:
Ok, I think it's not that big a deal if there's not a better option, but we might want to leave a follow up JIRA filed for when there is a better image to use. The Vagrantfile is just for testing, but I'm not sure everyone will want to grab an image from a domain they might not recognize.